Resources/views/base_uncompressed.html.twig

```diff
@@ -85,6 +85,9 @@
         {# Wrapper end #}
         </div>
 
+    {% block modals %}
+    {% endblock modals %}
+    
```

Comment:
base.html.twig extends base_uncompressed.html.twig, so I think adding to uncompressed is OK...